src/components/dropdown/Dropdown.tsx

```diff
@@ -187,6 +197,7 @@ export const Dropdown: FunctionComponent<DropdownProps & SpaceProps> = ({
           {options &&
             options.map((option: string | IOption) => (
               <DropdownItem
+                title={typeof option === "string" ? option : option.value}
```

Comment:
Why ?

Reply:
For better queries in testing where you can query by value and compare it to the expected without matching key-value pairs